core/src/main/AndroidManifest.xml

```diff
@@ -28,7 +28,7 @@
       android:name=".search.SearchActivity"
       android:windowSoftInputMode="adjustResize" />
 
-    <activity android:name=".bookmark.BookmarksActivity" />
+    <activity android:name="org.kiwix.kiwixmobile.core.page.bookmark.BookmarksActivity" />
```

Comment:
We should still be able to keep a relative path, alt enter and see if you get an option or just play around and delete a few segments